test/app-tap/datetime.test.lua

```diff
@@ -1316,6 +1351,27 @@ test:test("Time intervals creation - range checks", function(test)
         local err_msg, attribs = unpack(row)
         assert_raises(test, err_msg, function() return inew(attribs) end)
     end
+
+    local range_boundary = {
+        year = MAX_YEAR_RANGE,
+        month = MAX_MONTH_RANGE,
+        week = MAX_WEEK_RANGE,
+        day =  MAX_DAY_RANGE,
+        hour = MAX_HOUR_RANGE,
+        min = MAX_MIN_RANGE,
+        sec = MAX_SEC_RANGE,
+        nsec = MAX_NSEC_RANGE,
```

Comment:
Missed `MAX_USEC_RANGE` and `MAX_MSEC_RANGE`.
Both could be used in intervals:

```
tarantool>  datetime.interval.new{msec=2147}
---
- error: 'builtin/box/console.lua:415: value 2147 of msec is out of allowed range
    [-2147, 2147]'
...

tarantool>  datetime.interval.new{usec=2147}
---
- +0 seconds, 2147000 nanoseconds
...

tarantool> 
```

Reply:
Added